azure-devops/azext_devops/dev/team/_format.py

```diff
@@ -122,6 +122,35 @@ def _transform_team_member_row(row):
     return table_row
 
 
+def transform_users_table_output(result):
+    table_output = []
+    for item in result:
+        table_output.append(_transform_user_row(item))
+    return table_output
+
+
+def transform_user_table_output(result):
+    table_output = [_transform_user_row(result)]
+    return table_output
+
+
+def _transform_user_row(row):
+    table_row = OrderedDict()
+    table_row['ID'] = row['id']
+    if 'user' in row:
+        table_row['Display Name'] = row['user']['displayName']
+        table_row['Email'] = row['user']['mailAddress']
+    else:
+        table_row['Display Name'] = ' '
```

Comment:
why are we not setting `table_row['Email']` = '' in this case?

Reply:
Also just for my knowledge when would it happen that the email, access level, and status information will be missing?

Reply:
That check was for principalName. Switched to mail address, that's why not required now. Thanks